contrib/generate_tracy_header/header_top.txt

```diff
@@ -0,0 +1,36 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2022  The DOSBox Staging Team
```

Comment:
2022-2022, so it's easier to update later via scripts?

Reply:
Ahh yes, thanks for catching that!